geocode/geocode/src/main/java/at/wrk/geocode/poi/JsonPoi.java

```diff
@@ -1,33 +1,21 @@
 package at.wrk.geocode.poi;
 
-import at.wrk.geocode.Bounds;
-import at.wrk.geocode.LatLng;
-import at.wrk.geocode.ReverseResult;
-import at.wrk.geocode.autocomplete.AutocompleteKeyParser;
-import at.wrk.geocode.autocomplete.PreloadedAutocomplete;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import org.apache.commons.lang3.StringUtils;
-import org.apache.commons.lang3.time.StopWatch;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.core.io.Resource;
 import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
 
 import java.io.IOException;
 import java.io.InputStream;
-import java.time.Duration;
 import java.util.Arrays;
 import java.util.List;
-import java.util.Map;
-import java.util.TreeMap;
-import java.util.function.Function;
-import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 /**
- * Base class for providing POI entries from JSON resources
+ * Base class for providing POI entries from JSON resources in the "old" Coceso specific format
  */
-public abstract class JsonPoi extends PreloadedAutocomplete<Poi> implements PoiSupplier {
+public abstract class JsonPoi extends AbstractJsonPoi {
```

Comment:
As mentioned in the javadoc, how about "LegacyJsonPoi"?

Reply:
I thought about this, but it would require changes to additional modules (even though IDE refactoring should be able to handle that it would still be a breaking API change).